build/Targets/Settings.props

```diff
@@ -221,6 +231,10 @@
    -->
   <Import Project="$(ToolsetCompilerPropsFilePath)" Condition="Exists('$(ToolsetCompilerPropsFilePath)') AND '$(BootstrapBuildPath)' == '' AND '$(OS)' == 'Windows_NT'" />
 
+  <ItemGroup>
+    <PackageReference Include="XliffTasks" Version="$(XliffTasksVersion)" PrivateAssets="All" />
```

Comment:
Please don't add package reference elements at this level. It causes a number of complications in restore and can result in artifacts incorrectly being added to build output.

What is the goal here? Is it just to ensure that the package is restored during general build? If so please put this into ToolsetPackages\BaseToolset.csproj

Reply:
It needs to be restored, and every project (or at least every project that produces a shipping binary) needs to pick up its targets. Will that happen if I put it in ToolsetPackages\BaseToolset.csproj?

Reply:
Why does every project need the targets? 

Again: we assets a high degree of skepticism to global PackageReference because historically it has a 100% chance o f causing unwanted behavior in the future.

Reply:
I'll update this so we conditionally add the PackageReference for just shipping binaries. How does that sound?

Reply:
That sounds reasonable.